### ISSUE TITLE
Softaculous Fresh Install Export Error

### ISSUE THREAD

**Issue by yorkadam** [Telaxus/EPESI#231]
On a fresh install from Softaculous error occurs on export.

EPESI version 1.8.2 rev 20170701
Data directory is writeable
PHP version: 7.0.30 
Server: A2Hosting.com with brand new account with all the updated php and Softaculous versions.

Steps To Reproduce

1. Create a fresh install using Softaculous
2. Log into EPESI admin panel
3. View company to make sure it looks good
4. Go back to companies list
5. Click the EXPORT button at the top of the companies list

Expected Result - Export to occur
Actual Result - Error

> Type: E_USER_ERROR (256) Message: Invalid CommonData::get_array() request: System/csv_export_params File: /modules/Utils/CommonData/CommonDataCommon_0.php Line=253 error backtrace: [PHP core called function] function called: handle_epesi_error(256, "Invalid CommonData::get_array() request: System/csv_export_params", "/modules/Utils/CommonData/CommonDataCommon_0.php", 253, Array ("System/csv_export_params", "value", false, false, Array (#1), false)) File: /modules/Utils/CommonData/CommonDataCommon_0.php line 253 function called: trigger_error("Invalid CommonData::get_array() request: System/csv_export_params", 256) File: /modules/Utils/RecordBrowser/CsvExport.php line 36 function called: get_array("System/csv_export_params") File: /modules/Utils/RecordBrowser/csv_export.php line 31 function called: __construct("company", Object (Utils_RecordBrowser_Crits), Array (Array (#3)), "0") 

Note: this error is returned at the top of a blank page not at the bottom like other errors.

**Comment by georgehristov** [comment]
This error is caused by not installed `System/csv_export_params` array.

You can fix this but running following code under the admin mode >> Run PHP command

```
Utils_CommonDataCommon::new_array('System/csv_export_params'  ,array(
				'charset'              => 'UTF-8' //Charset in html/php style (for Polish CP1250)
				,'field_separator'      => ','      //field separator.
				,'decimal_separator'    => '.'      //decimal separator for float or currency
				,'end_line_type'        => 'UNIX'   //valid values are: WIN, WINDOWS, LIN, LINUX, UNI, UNIX, MAC, MACINTISH
				,'text_space_indicator' => 1        // add leading and ending separators to string witch spaces inside or not to add
				,'text_space_separator' => '"'      //leading and ending separator char for strings with spaces if 'text_space_indicator' = 1
		), true,false
		);
```

**Comment by yorkadam** [comment]
I just finished testing this and it worked. I tested the export button on other screens to confirm other exports work as expected and it does.  

I also wanted to add, for anyone in the future reading this, that to run the PHP as described (above) you might have to enable it by editing the config file in /data/config.php and adding  define('ALLOW_PHP_EMBEDDING', 1); so you will need to have permission on the host to edit PHP files.  

The Admin interface will let you know if it is enabled or not when you try to run it.  So, thanks to EPESI for that very nice and informative notice that explains exactly what to do. Awesome!

@georgehristov Thanks for your helpful reply that resolved the issue. I am very happy with the EPESI software. Thanks for an awesome product.
